log.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 VMware, Inc. or its affiliates. All Rights Reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package amqp091
+
+import (
+	"log"
+	"os"
+)
+
+var ErrLog = log.New(os.Stderr, "[ERROR] amqp091-go: ", log.LstdFlags|log.Lshortfile)
+var WarnLog = log.New(os.Stderr, "[WARNING] amqp091-go: ", log.LstdFlags|log.Lshortfile)
```

Comment:
Imho the package should not print anything to stdout/stderrr by default.
It is used as a library and it might mess up the output of the users application.
Also log messages of a package should only be needed in special cases for debugging.

Reply:
I agree